modelling/src/main/java/org/axonframework/modelling/configuration/ModellingConfigurer.java

```diff
@@ -27,15 +27,19 @@
 import org.axonframework.configuration.MessagingConfigurer;
 import org.axonframework.configuration.Module;
 import org.axonframework.configuration.ModuleBuilder;
+import org.axonframework.modelling.StateManager;
+import org.axonframework.modelling.HierarchicalStateManagerConfigurationEnhancer;
+import org.axonframework.modelling.annotation.InjectEntity;
 
 import java.util.function.Consumer;
 
 /**
- * The modelling {@link ApplicationConfigurer} of Axon Framework's configuration API, providing registration methods to, for
- * example, register a {@link StatefulCommandHandlingModule}.
+ * The modelling {@link ApplicationConfigurer} of Axon Framework's configuration API, providing registration methods to,
+ * for example, register a {@link StatefulCommandHandlingModule}.
  * <p>
- * This configurer does not set any defaults other than the defaults granted by the {@link MessagingConfigurer} it
- * wraps.
+ * This configurer is enhanced by the {@link HierarchicalStateManagerConfigurationEnhancer} to provide access to the
```

Comment:
Nice idea to add the enhancers given on this level! Perhaps we should do the same for the `AxonApplication`? :-)

Reply:
Sorry, fake news, this was an earlier version. It's now loaded by the service loader, so it's also automatically used by modules.